substratevm/src/com.oracle.svm.reflect/src/com/oracle/svm/reflect/serialize/SerializationSupport.java

```diff
@@ -121,6 +126,9 @@ public Object addConstructorAccessor(Class<?> declaringClass, Class<?> targetCon
 
     @Override
     public Object getSerializationConstructorAccessor(Class<?> declaringClass, Class<?> targetConstructorClass) {
+        if (Modifier.isAbstract(declaringClass.getModifiers()))
+            targetConstructorClass = stubConstructor.getDeclaringClass();
```

Comment:
This violates our codestyle. Please use
```java
if (Modifier.isAbstract(declaringClass.getModifiers())) {
    targetConstructorClass = stubConstructor.getDeclaringClass();
}
```

Reply:
With this changed the JavaDoc comment on `com.oracle.svm.reflect.serialize.SerializationSupport.StubForAbstractClass` is not valid anymore since it states:
_... So there won't be "new abstract class" bytecode anymore, and it's also safe for runtime as the corresponding newInstance method is never actually called._

For de-serializing the corresponding newInstance method for `StubForAbstractClass` will be called now.
Please update the comment accordingly.

Reply:
Forget what I said above. I ran @vjovanov s `HelloWorld` sample and verified that the JavaDoc comment is still holds.